src/styles/blocks/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo-nav {
+    display: flex;
+  }
+
+  &__logo {
+    display: block;
+  }
+
+  &-navigation {
+    display: flex;
+
+    &__list {
+      display: flex;
+      gap: 20px;
+      list-style-type: none;
+      text-transform: uppercase;
+      margin: 0;
+      padding: 0;
+    }
+
+    &__link {
+      display: flex;
+      font-size: 12px;
+      line-height: 60px;
+      text-decoration: none;
+      color: #060b35;
+
+      white-space: nowrap;
+      position: relative;
+      align-items: center;
+
+      &:hover {
+        color: #00acdc;
+        transition-duration: $transition__duration;
```

Comment:
The variable `$transition__duration` is used here but is not defined in this file. Ensure that this variable is defined in a file that is imported before this one, or define it at the top of this file.